data/tpc-h/workload_template.json

```diff
@@ -75,7 +75,187 @@
           },
           {
             "type": "expr",
-            "children": ["o_orderdate", "<", "date '$1' + interval '3' month"]
+            "children": ["o_orderdate", "<", "date '$2' + interval '3' month"]
+          }
+        ]
+      }
+    },
+    "5": {
+      "regex": "o_orderdate >= date '([0-9-]+)' and o_orderdate < date '([0-9-]+)' \\+ interval '1' year",
+      "predicate": {
+        "type": "and",
+        "children": [
+          {
+            "type": "expr",
+            "children": ["o_orderdate", ">=", "$1"]
+          },
+          {
+            "type": "expr",
+            "children": ["o_orderdate", "<", "date '$2' + interval '1' year"]
+          }
+        ]
+      }
+    },
+    "6": {
+      "regex": "l_shipdate >= date '([0-9-]+)' and l_shipdate < date '([0-9-]+)' + interval '1' year and l_discount >= '([0-9-]+)' and l_discount < '([0-9-]+)' and l_quantity < '([0-9-]+)'",
```

Comment:
You need to keep the `between ... and ...` for the regex to capture the code. You only need to transform it below.